core/controllers/reader.py

```diff
@@ -265,24 +264,22 @@ class PretestHandler(base.BaseHandler):
     @acl_decorators.can_play_exploration
     def get(self, exploration_id):
         """Handles GET request."""
-        start_cursor = self.request.get('cursor')
         story_id = self.request.get('story_id')
         story = story_fetchers.get_story_by_id(story_id, strict=False)
         if story is None:
             raise self.InvalidInputException
         if not story.has_exploration(exploration_id):
             raise self.InvalidInputException
-        pretest_questions, _, next_start_cursor = (
-            question_fetchers.get_questions_and_skill_descriptions_by_skill_ids(
+        pretest_questions = (
+            question_services.get_questions_by_skill_ids(
                 feconf.NUM_PRETEST_QUESTIONS,
                 story.get_prerequisite_skill_ids_for_exp_id(exploration_id),
-                start_cursor)
+                False)
```

Comment:
Would you mind changing the name of the third arg in this function to require_medium_difficulty? That seems to be what is actually implemented if I'm reading it right...

(And in that case I think it's fine to have this be True -- it make sense for all pre-test questions to be medium difficulty.)

Reply:
Done.